src/main/scala/cromwell/engine/backend/jes/JesEngineFunctions.scala

```diff
@@ -86,6 +87,14 @@ class JesEngineFunctionsWithoutCallContext extends WdlStandardLibraryFunctions w
  */
 class JesEngineFunctions(jesBackendCall: JesBackendCall) extends JesEngineFunctionsWithoutCallContext {
 
+  override protected def glob(params: Seq[Try[WdlValue]]): Try[WdlArray] = {
+    for {
+      singleArgument <- extractSingleArgument(params)
+      files = authenticated { _.storage.listContents(s"${jesBackendCall.callGcsPath}/globbed-${DigestionUtil.md5Sum(singleArgument.valueString)}") }
```

Comment:
The glob path looks strikingly like the one above, perhaps pull this out to a function?  Also there probably shouldn't be so much work happening in a string interpolation.